### PR TITLE
Functions - Modiji

### PR DESCRIPTION
**Pull request by Abduliitr**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [x] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [x] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [x] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [x] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Review by rpapavan97 — APPROVED** [review]
Code is fine, but as this problem is related to functions we can give some empty function and ask students to solve in it

**Comment by sailesh-ramayanam** [comment]
@rpapavan97 

1. Please add the template for function (as suggested by you)
2. The output for the sample test case seems wrong. Shouldn't it be `109` instead of `113`?

File: solutions/functions/modiji.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function calculateTotalFund(salaries, peopleCount){
+    let funds = 0;
+    for(let i = 0; i < peopleCount; i++){
+        funds += Math.ceil(0.07 * salaries[i]);
```
Comment (rpapavan97) [inline solutions/functions/modiji.js:17]: hi @Abduliitr 
if we are calculating the 7% of salary using the approach -> 0.07 * salaries[i], for sample salary of 100 we are getting the output as 7.0000001, which is incorrect and making ceil of this output we are getting the final output as 8 which is +1 of the correct output
but if we calculate using (7*salaries[i])/100) we are getting the correct output , and also even if we use pen and paper for the sample test case provided in problem description the output  should be 109, try to modify the code with the appropriate approach and commit it

**Review by rpapavan97 — CHANGES_REQUESTED** [review]

**Comment by rpapavan97** [comment]
@sailesh-ramayanam ,

1. Added the template for the problem
2. Changed the testcases output according to the calculation of input with correct approach

File: solutions/functions/modiji.js (hunk 1 above)
Comment (Abduliitr) [reply solutions/functions/modiji.js]: Updated

`        funds += Math.ceil((7*salaries[i])/100);
`

File: solutions/functions/modiji.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+function Convert_to_Number(list){
+    for(let each in list){
+        list[each]=Number(list[each])
+    }
+}
+
+let peopleCount = parseInt(readLine());
+let salaries = readLine().split(" ");
+Convert_to_Number(salaries);
```
Comment (sailesh-ramayanam) [inline solutions/functions/modiji.js:30]: @rpapavan97 Please rename `Convert_to_Number` to `camelCase`.

Comment (rpapavan97) [reply solutions/functions/modiji.js:30]: updated it

**Review by sailesh-ramayanam — APPROVED** [review]

**Comment by sailesh-ramayanam** [comment]
@rpapavan97 Test case explanation is missing in the problem. Please add it.

**Comment by rpapavan97** [comment]
added testcase explanation

**Review by rpapavan97 — APPROVED** [review]

**Pull request merged**